galata/test/jupyterlab/outputarea-stdin.test.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, IJupyterLabPageFixture, test } from '@jupyterlab/galata';
+
+const fileName = 'stdin.ipynb';
+
+async function populateNotebook(page: IJupyterLabPageFixture) {
+  await page.notebook.addCell('code', 'raise');
+}
+
+test.describe('Stdin for ipdb', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.notebook.createNew(fileName);
+  });
+
+  test('Stdin history search', async ({ page }) => {
+    await populateNotebook(page);
+    await page.notebook.addCell('code', '%debug');
+    await page.notebook.run();
```

Comment:
```suggestion
    await page.notebook.runCellByCell();
```

`raise` throws so `%debug` is not run. I think that `runCellByCell` should solve this.